github/repos.go

```diff
@@ -320,8 +320,9 @@ func (s *RepositoriesService) GetCodeOfConduct(ctx context.Context, owner, repo
 		return nil, nil, err
 	}
 
-	// TODO: remove custom Accept header when this API fully launches.
-	req.Header.Set("Accept", mediaTypeCodesOfConductPreview)
+	// TODO: remove custom Accept header when this API fully launches
+	acceptHeaders := []string{mediaTypeCodesOfConductPreview, mediaTypeGitHubAppsPreview}
+	req.Header.Set("Accept", strings.Join(acceptHeaders, ","))
```

Comment:
Up until this PR, we've been using `", "` (comma, followed by space) rather than `","` (just comma) as the header value separator... I think we should be consistent.

Did you want to drop the space on purpose? If it's valid (most likely, but should be tested if we don't want to take any chances), I suppose it's a little more space efficient, at the cost of being slightly less human readable.

In any case, given how big and hard to deal with this PR will be already, I'd propose you factor out this change into a separate PR, and keep this one as minimal as possible. So, use comma-space as before for consistency.

Reply:
Thanks! I'll just use `", "` for consistency.